spec/models/book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe 'Book' do
+  subject(:unLibro) { Book.new }
```

Comment:
This isn't PdeP!!!! Always in english! We usually call the subject as the model, in this case would be
```ruby
subject(:book) { Book.new }
```

But, instead of create a new instance manually, we could use [factory_bot](https://github.com/thoughtbot/factory_bot) with [faker](https://github.com/stympy/faker) to create Books with random data! Take a look to those gems and let me know if you need any help with the setup!